### DISCUSSION TITLE
hpool.com revenue only 60% of real earnings

### DISCUSSION THREAD

**Discussion by FSys77** [Chia-Network/chia-blockchain#4730] [General]
My revenue on hpool.com differs quite significantly from what I calculated via https://chiacalculator.com/
I adjusted the difficulty value on chiacalculator to reflect the fast increase in netspace.
Still I only get about 60% of what I should get.

Has anyone made similar experiences?

**Comment by ilyakooo0** [comment]
I think all pools are scams right now https://github.com/Chia-Network/chia-blockchain/discussions/4520

**Reply by SackofFluor** [reply]
Well... lets say they are not scams, but they seem to ask for a very high "express fee" from users that cant wait for them to farm any chia on their own

**Reply by Mareknown** [reply]
Here the same, around 60%

**Comment by oytp** [comment]
How can you calculate pool earnings lol the network is growing very fast, you cannot calculate FUTURE earnings. Just for the previous 24h as usual. Learn something about mining

**Reply by Mareknown** [reply]
Oooo!!! Somebody from Hpool??? The truth is that you get around 60% of XCH based on the calculation got from Chia calculator. So if somebody want to join Hpool and calculate the future profit, It should lower the results got in the Chia calculator by 60%. Of course the everyday profit is different depending on the space.

**Reply by FSys77** [reply]
> How can you calculate pool earnings lol the network is growing very fast, you cannot calculate FUTURE earnings. Just for the previous 24h as usual. Learn something about mining

I KNOW the netspace and difficulty of the of the PAST, hence I can, for example, calculate my expected earnings for the PREVIOUS 24 hours. The earnings from hpool are only about 60% of the expected earnings.

Did I say anything about calculating FUTURE earnings? No! So whats your point?

**Reply by oytp** [reply]
> > How can you calculate pool earnings lol the network is growing very fast, you cannot calculate FUTURE earnings. Just for the previous 24h as usual. Learn something about mining
> 
> I KNOW the netspace and difficulty of the of the PAST, hence I can, for example, calculate my expected earnings for the PREVIOUS 24 hours. The earnings from hpool are only about 60% of the expected earnings.
> 
> Did I say anything about calculating FUTURE earnings? No! So whats your point?

Because if you read something about mining, you cannot calculate even expectations. The network grows dramatically fast. Faster than values you use in your calculator

**Reply by FSys77** [reply]
> Because if you read something about mining, you cannot calculate even expectations. The network grows dramatically fast. Faster than values you use in your calculator

I am mining various coins since 2011.

I took the maximum difficulty value of the past 24 hours. That should give me the minimum earnings I should have gotten in the past 24 hours. hpool was only 60% of that.

So the pool is a rip off. Simple as that.

**Reply by oytp** [reply]
I know what you mean, but also exclude some plots you didn't have yesterday. For me the difference isn't so huge, only 15-20%, but yes, it isn't PPLNS 0% :D

Anyway, UUPOOL gives you less, I did compare that. And anyway again, we now have only 2 options: 1) Solo 2) Pool (H or UU - you decide). So if you want to have something to touch and spend, pools are the only option for now.
